gemini.go

```diff
@@ -34,31 +34,48 @@ func GeminiImage(imgData []byte) (string, error) {
 	return printResponse(resp), nil
 }
 
-// Gemini Chat Complete: Iput a prompt and get the response string.
-func GeminiChatComplete(req string) string {
-	ctx := context.Background()
-	client, err := genai.NewClient(ctx, option.WithAPIKey(geminiKey))
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer client.Close()
-	model := client.GenerativeModel("gemini-pro")
-	model.Temperature = 0.3
+func startNewChatSession() *genai.ChatSession {
+    ctx := context.Background()
+    client, err := genai.NewClient(ctx, option.WithAPIKey(geminiKey))
+    if err != nil {
+        log.Fatal(err)
+    }
+    model := client.GenerativeModel("gemini-pro")
+    model.Temperature = 0.3
 	cs := model.StartChat()
+    // init prompt
```

Comment:
Do we still need those comment codes?

Reply:
You are right, we don't need to comment out the part for now, I will submit again